docs/index.asciidoc

```diff
@@ -21,6 +21,16 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+If you plan to use the Kibana web interface, use the Elasticsearch output plugin
+to get your log data into Elasticsearch. 
+
+This output only speaks the HTTP protocol. HTTP is the preferred protocol for interacting with Elasticsearch.
```

Comment:
We can remove this paragraph as node protocol hasn’t been available for some time and we don’t provide transport protocol support anymore in Logstash.